### DISCUSSION TITLE
Allow Access to All Photos Button

### DISCUSSION THREAD

**Discussion by Weeli-MC** [appium/appium#19882] [Q&A]
Hi Guys!

I am trying to get the test to click on the "Allow Access to All Photos" but I can't. Anyone has any solution for this?  I've tried many solutions and the simulator cant find the element: `  await $(
            `//XCUIElementTypeButton[@name=“Allow Access to All Photos”]`,
        ).click();`.  `        "**/XCUIElementTypeSheet/**/XCUIElementTypeButton[`label == 'Allow Access to All Photos'`]",`.   ` ("-ios predicate string:type == "XCUIElementTypeButton" AND label == "Allow Access to All Photos" AND name == "Allow Access to All Photos"")  `.    

Please help! Thank you.   

<img width="255" alt="Screenshot 2024-03-12 at 12 07 08 AM" src="https://github.com/appium/appium/assets/155956864/4ac59a50-a0ef-4c20-b50d-948870b2247d">

**Comment by KazuCocoa** [comment]
https://appium.github.io/appium-xcuitest-driver/latest/guides/troubleshooting/#interact-with-dialogs-managed-by-comapplespringboard would help.
